src/components/PanelMenuWrapper.js

```diff
@@ -53,16 +56,28 @@ class PanelsWithSidebar extends Component {
       group = child.props.group;
       name = child.props.name;
 
-      if (groupLookup.hasOwnProperty(group)) {
-        groupIndex = groupLookup[group];
-        obj = groups[groupIndex];
-      } else {
+      if (child.type.name === 'Button') {
```

Comment:
Are we expecting this always to be our `Button` type? If so we can compare classes instead of child.type.name. From Section.js

```
if (child.type === Info) {
...
}
```

Do we need to restrict users to our `Button`? Is there a way to allow users to pass any arbitrary component to be added to the Sidebar.

What if we had a `SidebarItem` component that wraps whatever you want to include in the sidebar?

```jsx
<PanelMenuWrapper >
   <SidebarItem name="save">
       <a onClick={saveMyApp} />
    </SidebarItem>
</PanelMenuWrapper>
```
This way users can put any thing in a `SidebarItem` container and we just render it.